operations/mimir-mixin-compiled/dashboards/mimir-overview.json

```diff
@@ -163,7 +163,7 @@
             "height": "250px",
             "panels": [
                {
-                  "content": "These panels show an overview on the write path. \nVisit the following specific dashboards to drill down into the write path:\n\n- <a target=\"_blank\" href=\"./d/8280707b8f16e7b87b840fc1cc92d4c5/mimir-writes?${__url_time_range}&${__all_variables}\">Writes</a>\n- <a target=\"_blank\" href=\"./d/bc9160e50b52e89e0e49c840fea3d379/mimir-writes-resources?${__url_time_range}&${__all_variables}\">Writes resources</a>\n- <a target=\"_blank\" href=\"./d/978c1cb452585c96697a238eaac7fe2d/mimir-writes-networking?${__url_time_range}&${__all_variables}\">Writes networking</a>\n",
+                  "content": "These panels show an overview on the write path. \nVisit the following specific dashboards to drill down into the write path:\n\n- <a target=\"_blank\" href=\"./d/8280707b8f16e7b87b840fc1cc92d4c5/mimir-writes?${__url_time_range}&${__all_variables}\">Writes</a>\n- <a target=\"_blank\" href=\"./d/bc9160e50b52e89e0e49c840fea3d379/mimir-writes-resources?${__url_time_range}&${__all_variables}\">Writes resources</a>\n- <a target=\"_blank\" href=\"./d/978c1cb452585c96697a238eaac7fe2d/mimir-writes-networking?${__url_time_range}&${__all_variables}\">Writes networking</a>\n- <a target=\"_blank\" href=\"./d/a9b92d3c4d1af325d872a9e9a7083d71/mimir-overview-resources?${__url_time_range}&${__all_variables}\">Overview resources</a>\n",
```

Comment:
This is problematic because "writes" is a verb and a noun in this context. Is it "Writes’ resources"? or is it "[Something] writes resources"?

Reply:
My impression is that this is multiple _writes’_ resources rather than a single _write’s_ resources.

Reply:
Please simplify the multiple prepositions (down into) in "Visit the following specific dashboards to drill down into the write path:" Suggestion: `To examine the write path in detail, see a specific dashboard:`

Reply:
See: https://github.com/grafana/mimir/pull/3488